src/main/java/net/snowflake/client/util/DecorrelatedJitterBackoff.java

```diff
@@ -19,4 +20,15 @@ public DecorrelatedJitterBackoff(long base, long cap) {
   public long nextSleepTime(long sleep) {
     return Math.min(cap, ThreadLocalRandom.current().nextLong(base, sleep * 3));
   }
+
+  public long getJitterForLogin(long currentTime) {
+    int mulitplicationFactor = chooseRandom(-1, 1);
+    long jitter = (long) (mulitplicationFactor * currentTime * 0.5);
+    return jitter;
+  }
+
+  private int chooseRandom(int min, int max) {
+    Random random = new Random();
+    return random.nextInt(max - min) + min;
```

Comment:
Since we call chooseRandom(-1, 1), with nextInt we basically only get -1, 0, 1 as possible return value? I would expected more random on the jitter. Like if the base backoff time is 32, the final jitter timeout could be any number between 16~48.

Reply:
I updated it now so it can actually be any value between (-1, 1) for getting the random multiplier.